client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/style.scss

```diff
@@ -174,38 +170,52 @@ h2.action-panel__title {
 }
 
 .email-provider-stacked-card__title-price-badge {
-	width: $width-right-panel-card;
+	width: 100%;
+	margin-left: 10px;
+	@include break-small {
+		width: $width-right-panel-card;
+	}
 }
 
 .email-provider-stacked-card__provider-form-and-right-panel {
 	display: none;
 }
 
+.email-providers-stacked-comparison__main.main .card.action-panel.promo-card.email-providers-stacked-comparison__provider-card .action-panel__figure {
+	margin-right: unset;
+	@include break-mobile {
+		margin-right: 24px;
+	}
+}
+
+.email-provider-stacked-card__provider-expand-cta {
+	width: 110px;
+}
+
 .is-expanded {
 	.email-provider-stacked-card__provider-expand-cta {
 		display: none;
 	}
 
 	.email-provider-stacked-card__provider-form-and-right-panel {
 		display: flex;
-		flex-direction: column-reverse;
+		flex-direction: column;
 		justify-content: start;
 		margin-top: 1em;
 		line-height: 1.5;
-		padding-right: 20px;
 
-		@include breakpoint-deprecated( '>1040px' ) {
+		@include break-mobile {
 			flex-direction: row;
 		}
 	}
 
 	.email-provider-stacked-card__provider-form {
 		margin-bottom: 20px;
-		margin-right: 20px;
-		width: $width-left-panel-card;
+		width: 100%;
 
-		@include breakpoint-deprecated( '>1040px' ) {
-			margin-top: 0;
+		@media screen and ( min-width: 481px ) {
+			min-width: $width-left-panel-card;
+			max-width: $width-left-panel-card;
 		}
```

Comment:
Two notes:
1. I think this breakpoint is too narrow -- this doesn't look great at widths between 480px and ~630px. I think we should flip to a horizontal layout at much wider widths. (We likely need the `.email-provider-stacked-card__provider-form>* .form-text-input-with-affixes` selector to use similar breakpoints.)
<img width="523" alt="Screenshot 2021-12-14 at 10 24 54" src="https://user-images.githubusercontent.com/3376401/145960871-aeb3c149-b1fc-4804-8e16-dd2b9c9800cd.png">
2. Why are we using a hard-coded breakpoint of 481px here? We should be using a value like `@include break-mobile`.

Reply:
The only thing I has been really struggling to make it work is to fit mailbox-domain input-span. I have added some max sizes but I am not sure if I have achieved the right solution as I couldn't limit the space of the flex container to the parent HTML element 🙁